README.md

```diff
@@ -1,4 +1,6 @@
-# Snail-Kotlin
+# Snail-Kotlin 🐌 ![CircleCI](https://circleci.com/gh/UrbanCompass/Snail-Kotlin/tree/master.svg?style=shield&circle-token=02af7805c3430ec7945e0895b2108b4d9b348e85) [![codecov](https://codecov.io/gh/UrbanCompass/Snail-Kotlin/branch/master/graph/badge.svg)](https://codecov.io/gh/UrbanCompass/Snail-Kotlin)
+
+A light-weight observables framework, also available in [Swift](https://github.com/UrbanCompass/Snail)
 
 Download
```

Comment:
could you fix this too? 
![screen shot 2017-03-06 at 10 17 43 am](https://cloud.githubusercontent.com/assets/6198867/23615959/29a5ead2-0256-11e7-9b7c-74fa5a8a5cb7.png)